src/vdaf/prio3.rs

```diff
@@ -83,27 +83,28 @@ impl Prio3Count {
 /// The count-vector type. Each measurement is a vector of integers in `[0,2^bits)` and the
 /// aggregate is the element-wise sum.
 pub type Prio3SumVec =
-    Prio3<SumVec<Field128, ParallelSum<Field128, BlindPolyEval<Field128>>>, XofShake128, 16>;
+    Prio3<SumVec<Field128, ParallelSum<Field128, Mul<Field128>>>, XofShake128, 16>;
 
 impl Prio3SumVec {
     /// Construct an instance of Prio3SumVec with the given number of aggregators. `bits` defines
     /// the bit width of each summand of the measurement; `len` defines the length of the
     /// measurement vector.
-    pub fn new_sum_vec(num_aggregators: u8, bits: usize, len: usize) -> Result<Self, VdafError> {
-        Prio3::new(num_aggregators, SumVec::new(bits, len)?)
+    pub fn new_sum_vec(
```

Comment:
The spec leaves this up to the user. This is because the optimal choice is kind of complicated: https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-vdaf#parallel-sum-chunk-length

Unfortunately I think we have to expose this to the user, in order to support interop with other VDAF implementations. (The prover and verifier need to agree on the value.)

Reply:
Yes, see [section 7.4.3.1 of the VDAF editor's draft](https://cfrg.github.io/draft-irtf-cfrg-vdaf/draft-irtf-cfrg-vdaf.html#section-7.4.3.1). Prior to this PR, we used the "square root of the encoded measurement length" heuristic, which is asymptotically correct, but not optimal when you look at concrete sizes. We didn't want to codify that heuristic into the VDAF spec, both because we'd have to specify a particular way to compute and round an integer square root, and because that would often be a suboptimal chunk length source (with a worst case penalty of ~50%). Since the chunk length affects circuit calculations, we need all parties to agree on it, so it's pushed up to be a VDAF parameter.